frontend/src/metabase/admin/settings/components/widgets/PublicLinksListing/PublicLinksListing.jsx

```diff
@@ -174,24 +176,32 @@ export const PublicLinksActionListing = connect(mapStateToProps)(
   },
 );
 
-export const EmbeddedDashboardListing = () => (
-  <div className="bordered rounded full" style={{ maxWidth: 820 }}>
-    <PublicLinksListing
-      load={DashboardApi.listEmbeddable}
-      getUrl={dashboard => Urls.dashboard(dashboard)}
-      type={t`Embedded Dashboard Listing`}
-      noLinksMessage={t`No dashboards have been embedded yet.`}
-    />
-  </div>
-);
-
-export const EmbeddedQuestionListing = () => (
-  <div className="bordered rounded full" style={{ maxWidth: 820 }}>
-    <PublicLinksListing
-      load={CardApi.listEmbeddable}
-      getUrl={question => Urls.question(question)}
-      type={t`Embedded Card Listing`}
-      noLinksMessage={t`No questions have been embedded yet.`}
-    />
-  </div>
+export const EmbeddedResources = () => (
+  <Stack spacing="md" className="flex-full">
+    <div>
+      <Text mb="sm">{t`Embedded Dashboards`}</Text>
+      <div className="bordered rounded full" style={{ maxWidth: 820 }}>
+        <PublicLinksListing
+          data-testId="-embedded-dashboards-setting"
```

Comment:
We don't seem to use `-embedded-dashboards-setting` or `-embedded-questions-setting` anymore. Can you remove these `data-testId`s that also means we can revert the change at line 68 too?

Reply:
They're used _dynamically_ here: https://github.com/metabase/metabase/blob/9be81f36ade0249fa280e73c78b6c51153a45b5b/e2e/test/scenarios/embedding/embedding-smoketests.cy.spec.js#L212-L214

I tried to make it use findByText.closest etc but it was becoming dirtier than leaving the original test-ids

Reply:
Well this is very hard to find. I'm fine leaving `data-testId` as is, but could you help fix `[embedding-smoketests.cy.spec.js](https://github.com/metabase/metabase/blob/9be81f36ade0249fa280e73c78b6c51153a45b5b/e2e/test/scenarios/embedding/embedding-smoketests.cy.spec.js#L212-L214)` so it doesn't use the ID dynamically? We should never create any ID dynamically as it's very hard to find and replace, and makes migrating and refactoring close to impossible.

Reply:
> Well this is very hard to find.
💯, I spent quite some time yesterday because of that, I should have pointed that out with a comment on the PR.

I updated the tests to use an inline object to get the full test id so that's cmd+f friendly